src/templates/_includes/tabs.twig

```diff
@@ -8,38 +8,37 @@
         class: (tab.class ?? [])|explodeClass,
 })) %}
 
-{% tag 'nav' with containerAttributes %}
-    <ul class="scrollable">
+{% tag 'div' with containerAttributes %}
+    <div class="scrollable" role="tablist" aria-label="{{ 'Fields'|t('app') }}">
```

Comment:
@brandonkelly targeted this for the Entry page, but where's would one pass an appropriate label depending on the template?

Reply:
If we want this to be configurable, you could use a new `tabContainerLabel` variable:

```twig
aria-label="{{ tabContainerLabel ?? 'Fields'|t('app')"
```